canonical-web-and-design/ubuntu.com
# WIP: Build homepage

## Done

- Build the homepage based on the [copy doc](https://docs.google.com/document/d/1ySJxQbqVdeH4Tra0zwBm2Tn0s56kFGnEF7d8xDRTxwU/edit) and [design](https://github.com/canonical-web-and-design/ubuntu.com/issues/7969)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure that the content matches the copy doc
- Ensure that the code is sensible and in line with any Canonical style guides
- Check that the layout of the page is suitable - there are several instances where Vanilla's default behaviour did not give 
a 1:1 correspondence with the design


## Issue / Card

Fixes #7970
